web/src/main/java/co/mobileaction/example/web/repository/IPostRepository.java

```diff
@@ -18,4 +19,6 @@ public interface IPostRepository extends JpaRepository<Post, Long>
    @Transactional
    void deleteByUserId(Long userId);
     List<Post> findAllByUserId(Long userId);
+    @Query("select distinct p.userId from Post p where p.userId is not null")
+    java.util.List<Long> findDistinctUserIds();

```

Comment:
You should import java.util.List in the class and write the method return parameter as below:
```suggestion
    List<Long> findDistinctUserIds();
```